test/fixtures/templater/jinja_exclude_macro_path/macros/macro_include.sql

```diff
@@ -0,0 +1,3 @@
+{%- macro foo1() -%}101{%- endmacro -%}
+
+{%- macro foo2() -%}102{%- endmacro -%}
```

Comment:
Can you change these macros to be different from the ones in the `.sqlfluff` to demonstrate the precedence between competing definitions? Potentially consider removing the definitions from the `.sqlfluff` file to make this a cleaner test of just the overrides.